Hey, thanks for contribution. This is good first start :). Basically, you added example for sub-package and it's also good thing. The best would be to add example to utility methods from `steps.go` file. 

Goal here is to show any developer how real world usages of library can looks like.
Steps required to properly use library are:
1) Initialize [APIContext](https://github.com/pawelWritesCode/gdutils/blob/master/context.go#L19). We can use one of 2 constructors, easier path is to use [this](https://github.com/pawelWritesCode/gdutils/blob/master/context.go#L81) one
2) Now, we have plenty of methods to work on from [steps.go](https://github.com/pawelWritesCode/gdutils/blob/master/steps.go) file.

Let's egzamine [save](https://github.com/pawelWritesCode/gdutils/blob/master/steps.go#L1382) method, which allows user to save any data into scenario cache. Let's say, we would like to save as `HUMAN_AVG_TEMP`, value `36.6`. 

So example logic may look like this (i write it from hand, so may contain errors):
```
ac := NewDefaultAPIContext(false, "")
if err := ac.Save("HUMAN_AVG_TEMP",  36.6); err != nil {
     //log error somehow
}

temp, err := ac.Cache.GetSaved("HUMAN_AVG_TEMP")
if err != nil {
     //log error somehow
}

fmt.Println(temp)
// Output: [36.6]
```

Each method from `steps.go` returns `error` interface. The logic of examples may trigger error path on method (return not nil error) and print error string (so we can write our assertion. for example: `// Output: [97]`, as you did). There are similar examples in standard go library. 

Tell me whether you want to try to add another examples, or you just end on this. 

PS: `go vet` returns errors, please repair them: 
```
Error: ./steps_test.go:94:1: ExampleRunesFromCharSet refers to unknown identifier: RunesFromCharSet
Error: Process completed with exit code 2.
```

--- a/steps_test.go
+++ b/steps_test.go
@@ -91,6 +91,11 @@ func (m *mockedJsonPathFinder) Find(expr string, jsonBytes []byte) (any, error) 
 	return args.Get(0).(any), args.Error(1)
 }
 
+func ExampleRunesFromCharset() {
+	fmt.Println(stringutils.RunesFromCharset(1, []rune("a")))
+	// Output: [97]
+}
+
 func TestState_RequestPrepare(t *testing.T) {
 	type args struct {
 		method      string

